CommonMark/Syntax/InlineTag.cs

```diff
@@ -1,13 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace CommonMark.Syntax
 {
     /// <summary>
     /// Specifies the element type of an <see cref="Inline"/> instance.
     /// </summary>
-    public enum InlineTag
+    public enum InlineTag : byte
```

Comment:
I think I tried setting these enums as based on bytes but that did not improve anything. If I remember correctly, it might even have had the opposite effect (but even if it was so, it was negligible).